eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,93 @@
+$Language = "rust"
+$LanguageDisplayName = "Rust"
+$PackageRepository = "crates.io"
+$packagePattern = "Cargo.toml"
+$MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/main/_data/releases/latest/rust-packages.csv"
```

Comment:
This doesn't exist yet but probably won't be needed until we start working on docs.

Reply:
@weshaggard, should I drop that line or make an empty file?

Reply:
I would at least comment this out until a time it is consumed.

Reply:
commented